### ISSUE TITLE
[Breaking change] Remove legacy templated parent in Uno 6

### ISSUE THREAD

**Issue by Youssef1313** [unoplatform/uno#18672]
### What would you like to be added

@Xiaoy312 Is the legacy templated parent code is intended to be removed in Uno 6?

In that case, that should be tracked in https://github.com/unoplatform/uno/issues/8339

**Comment by Xiaoy312** [comment]
@jeromelaban we should be able to safely remove the now obsolete legacy TemplatedParent dp with uno6, right?
but what about the codegen? do we keep that in the ~~temporary~~permanent solution of DOStoreGenerator, or move it to DOGenerator as initially planned?

**Comment by jeromelaban** [comment]
Yes, we'll remove that in the next build. As for the DOStore, if there's a performance impact, we could change it for the fastest one. We'll advise at that time.
